tests/utils/dockercli/volumelifecycle.go

```diff
@@ -53,8 +65,35 @@ func KillDocker(ip string) ([]byte, error) {
 	return out, err
 }
 
+// KillVDVSPlugin - kill docker daemon. It is restarted automatically
+func KillVDVSPlugin(ip string) ([]byte, error) {
+	log.Printf("Killing vDVS plugin on VM [%s]\n", ip)
+	out, err := ssh.InvokeCommand(ip, dockercli.GetVDVSPlugin)
+	if err != nil {
+		return out, err
+	}
+
+	pluginID := strings.Fields(misc.FormatOutput(out))[0]
+	out, err = ssh.InvokeCommand(ip, dockercli.KillVDVSPlugin+pluginID)
+
+	time.Sleep(2 * time.Second)
```

Comment:
keeping it here for now.  can move it if need comes to have a control over this wait time.